src/stability_sdk/animation.py

```diff
@@ -852,8 +851,10 @@ def transform_video(self, frame_idx) -> Optional[np.ndarray]:
                 # warp_flow is in `extras` and will change in the future
                 prev_b64 = base64.b64encode(image_to_png_bytes(self.video_prev_frame)).decode('utf-8')
                 next_b64 = base64.b64encode(image_to_png_bytes(video_next_frame)).decode('utf-8')
-                extras = { "warp_flow": { "prev_frame": prev_b64, "next_frame": next_b64} }
-                transformed_prior_frames, mask = self.api.transform(self.prior_frames, None, extras=extras)
+                extras = { "warp_flow": { "prev_frame": prev_b64, "next_frame": next_b64, "export_mask": args.inpaint_border } }
```

Comment:
I'd consider making `export_mask` its own arg independent from `inpaint_border` (.. and try to make sure it's passed in extras in other scenarios as well), since it's something we might like to play with in development and could find other uses (plus it could lead to an easier-to-read code structure). There's no need to do that now though.

Reply:
Yeah currently the only time we want masks returned is for doing inpainting but it would be nice to have some debug visualization modes which would want this independently.